static/docs/user-guide/using-docker.md

```diff
@@ -32,7 +32,7 @@ you can take a look at https://pythonspeed.com/docker/
 Since `dvc` is a command line application, we have several recommendiations
 to enhance the experience:
   * Make sure your _locale_ is set to UTF-8
-  * Install the `Zsh` shell and its respective autocompletion script.
+  * Install the `zsh` shell and its respective autocompletion script.
```

Comment:
Looks like Zsh is correct) I'll remove the backticks.
https://en.wikipedia.org/wiki/Z_shell